tests/coafileTest.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import re
+from coalib.misc.Shell import run_shell_command
+from coalib.settings.ConfigurationGathering import load_config_file
+
+
+class coafileTest(unittest.TestCase):
+
+    def setUp(self):
+        self.filepath = '.coafile'
+        self.sections = load_config_file(self.filepath, None, True)
+
+    def test_commit_regex(self):
+        shortlog_regex = self.sections['commit']['shortlog_regex'].value
+        git_command = 'git log -100 --pretty=%s'
+        git_log, stderr = run_shell_command(git_command)
+        self.assertEqual(stderr, '', msg=stderr)
+        commit_messages = git_log.splitlines()
+        for message in commit_messages:
```

Comment:
Why don't you use git_log.splitlines() directly here? Imo, it's shorter and still clear enough